### PR TITLE
feat: build correct footer into About page

### PR DESCRIPTION
**Pull request by just-toby**
build the new footer into the Landing/About page.

https://www.figma.com/file/TBW3lYklHnv7f7Kg9ajlHb/.org-v2?node-id=2166%3A14402&t=r0v3eB6pHbfVbuQh-0

https://uniswaplabs.atlassian.net/browse/WEB-2675?atlOrigin=eyJpIjoiMjhhNDhjYjk5MTM3NDY1M2E5YTZhZjQ3MTFhODFhOTIiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by JFrankfurt** [comment]
Can you add a link to the design for this?

**Comment by just-toby** [comment]
> Can you add a link to the design for this? (surprised it's not on the jira ticket)

https://www.figma.com/file/TBW3lYklHnv7f7Kg9ajlHb/.org-v2?node-id=2166%3A14402&t=r0v3eB6pHbfVbuQh-0

**Comment by JFrankfurt** [comment]
image

It looks like the top/bottom margin here should be 96px and 144px respectively

**Comment by JFrankfurt** [comment]
image

I'm a little confused about the `about` link here. There's not an active state, and it links to a different page than the one you land on by default, but it doesn't trigger refresh, nav, or active state once it's clicked on and the page goes to its url. It feels broken, but I get why it does this. Not sure if it should get a different design treatment or behavior. cc @infredible what do you think?

**Review by JFrankfurt — APPROVED** [review]

File: src/components/About/Icons.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components/macro'
+
+import { ReactComponent as DiscordI } from './images/discord.svg'
+import { ReactComponent as GithubI } from './images/github.svg'
+import { ReactComponent as TwitterI } from './images/twitter-safe.svg'
```
Comment (JFrankfurt) [inline src/components/About/Icons.tsx:5]: double check we don't already have these icons

**Review by JFrankfurt — DISMISSED** [review]
image
The unicorn and footer nav shouldn't be here on mobile

**Comment by just-toby** [comment]
gonna try to start a discussion ƒor the "about" link in the figma

**Review by JFrankfurt — APPROVED** [review]

**Pull request merged**